internal/yggdrasil/yggdrasil.go

```diff
@@ -448,6 +448,7 @@ func (h *Handler) toWorkloadList(ctx context.Context, logger logr.Logger, deploy
 
 		workload := models.Workload{
 			Name:          deployment.Name,
+			Namespace:     deployment.Namespace,
```

Comment:
I'm happy with the change, but I think that it'll be good to add at least one assert on the integration test :-)